include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -197,6 +197,49 @@ namespace internal
 {
   namespace DoFAccessorImplementation
   {
+    /**
+     * Convert an FE index that might contain the right value but also
+     * invalid_fe_index to a right value if needed/possible.
+     */
+    template <int structdim, int dim, int spacedim, bool level_dof_access>
+    unsigned int
+    get_fe_index(
```

Comment:
This kinds of functions are often called `*_or_something()`. A couple of recent additions to the standard are https://en.cppreference.com/w/cpp/utility/optional/value_or and https://en.cppreference.com/w/cpp/utility/optional/or_else

My suggestion would be to call this function `get_fe_index_or_default()` instead.

Reply:
I have rename it to `get_fe_index_or_default()`.